### PR TITLE
Integrate meta ai and other changes from LSC DC demo

### PR DESCRIPTION
**Pull request by jimmytyyang**
- demo branch in DC
- meta ai integration
- use skill manager for lsc

### CODE REVIEW

File: spot_rl_experiments/spot_rl/utils/whisper_translator.py [hunk 1]
```diff
@@ -49,6 +49,7 @@ def __init__(self):
         # Voice Activity Detection
         self.vad = webrtcvad.Vad()
         self.vad.set_mode(3)
+        self.whisper = whisper.load_model("base", device="cuda")
```
Comment (jimmytyyang) [inline spot_rl_experiments/spot_rl/utils/whisper_translator.py:52]: Tushar:

Change this based on online or offline loading!

File: bd_spot_wrapper/spot_wrapper/spot.py [hunk 2]
```diff
@@ -916,11 +918,17 @@ def dock(self, dock_id: int = DOCK_ID, home_robot: bool = False) -> None:
             dock_id: The dock to dock to
             home_robot: Whether to reset home the robot after docking
         """
-        blocking_dock_robot(self.robot, dock_id)
+        try:
+            blocking_dock_robot(self.robot, dock_id)
+        except Exception as e:
```
Comment (KavitShah1998) [inline bd_spot_wrapper/spot_wrapper/spot.py:923]: @KavitShah1998 
Remove exception handling

File: spot_rl_experiments/spot_rl/envs/lsc_demo.py [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+        print("PARSED", nav_1, pick, nav_2)
+
+        # Find closest nav_targets to the ones robot knows locations of
+        nav_1 = sentence_similarity.get_most_similar_in_list(
+            nav_1, list(waypoints_yaml_dict["nav_targets"].keys())
+        )
+        nav_2 = sentence_similarity.get_most_similar_in_list(
+            nav_2, list(waypoints_yaml_dict["nav_targets"].keys())
+        )
+        print("MOST SIMILAR: ", nav_1, pick, nav_2)
```
Comment (KavitShah1998) [inline spot_rl_experiments/spot_rl/envs/lsc_demo.py:65]: Add safety net

File: spot_rl_experiments/spot_rl/envs/lsc_demo.py [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+        # Do LSC by calling skills
+        slow_donw = 0.5  # slow down time to increase the skill execution stability
+        spot.nav(nav_1)
+        time.sleep(slow_donw)
+        spot.pick(pick)
+        time.sleep(slow_donw)
+        spot.nav(nav_2)
+        time.sleep(slow_donw)
+        spot.place(nav_2)
+        time.sleep(slow_donw)
```
Comment (KavitShah1998) [inline spot_rl_experiments/spot_rl/envs/lsc_demo.py:87]: fix type

File: spot_rl_experiments/spot_rl/llm/src/conf/config.yaml [hunk 5]
```diff
@@ -2,4 +2,5 @@ defaults:
   - llm: openai
   - prompt: rearrange_easy_few_shot
 
+llm_type: metaailocal # it could be "openai", "metaai", "metaailocal"
```
Comment (KavitShah1998) [inline spot_rl_experiments/spot_rl/llm/src/conf/config.yaml:5]: let's keep default as `metaai` ?

Comment (KavitShah1998) [inline spot_rl_experiments/spot_rl/llm/src/conf/config.yaml:5]: Do we need both `llm` & `llm_type`?

Comment (jimmytyyang) [reply spot_rl_experiments/spot_rl/llm/src/conf/config.yaml:5]: fixing this in the second PR. now llm flag is the actual file name

Comment (jimmytyyang) [reply spot_rl_experiments/spot_rl/llm/src/conf/config.yaml:5]: https://github.com/facebookresearch/spot-sim2real/issues/168

**Review by KavitShah1998 — APPROVED** [review]
LGTM

**Pull request merged**